hnn_core/neuron.py

```diff
@@ -406,9 +406,11 @@ def _create_all_spike_sources(self):
                             target_cell_type=target_cell_type,
                             params=self.net.p_unique[src_type],
                             gid=gid))
-                _PC.cell(gid,
-                         self.unique_feeds[src_type][-1].connect_to_target(
-                             self.net.params['threshold']))
+                _PC.cell(
+                    gid,
+                    self.unique_feeds[src_type]
+                    [-1].connect_to_target(
+                        self.net.params['threshold']))
```

Comment:
my flake8 checker needs to be updated I guess? It was complaining about this line

Reply:
Interesting. My checker only complained about the first alignment, but let me mix vertical alignment at `(` and hanging indent schemes. This is better.